lib/benchee/formatters/csv/general.ex

```diff
@@ -0,0 +1,63 @@
+defmodule Benchee.Formatters.CSV.General do
```

Comment:
I believe `General` is an unfortunate name as it is just so... general. I think calling it `Statistics` vs. `Raw` woould help distinguish them :)

Reply:
yep, makes sense.